pkg/apis/postgres-operator.crunchydata.com/v1beta1/pgbouncer_types.go

```diff
@@ -73,6 +73,10 @@ type PGBouncerPodSpec struct {
 	// +optional
 	Config PGBouncerConfiguration `json:"config,omitempty"`
 
+	// Defines custom sidecars for the pgBouncer pods
```

Comment:
🔧 Capitalize `PgBouncer`.
⛏️ Other fields in this section are phrased differently.

🤔 Changes here become changes to the pod spec which causes a rollout, right? If so, we should mention PgBouncer restart.

```suggestion
	// Custom sidecars for a PgBouncer pod.
	// Changes ... cause PgBouncer to restart.
	// Changing this ... causes PgBouncer to restart.
```

There's a variety of this phrase. Singular, plural, noun, gerund... 🤷🏻

Reply:
:+1: Updated as recommended, and I picked what seemed to be the most common option for the rollout warning.

Reply:
:thinking: This same behavior is true of the Postgres instance sidecars as well. I'm going to update that comment as well (as a separate commit).